src/main/java/graphql/validation/rules/DeferDirectiveLabel.java

```diff
@@ -0,0 +1,41 @@
+package graphql.validation.rules;
+
+import graphql.Internal;
+import graphql.language.Argument;
+import graphql.language.Directive;
+import graphql.language.Node;
+import graphql.language.StringValue;
+import graphql.validation.AbstractRule;
+import graphql.validation.ValidationContext;
+import graphql.validation.ValidationErrorCollector;
+import java.util.LinkedHashSet;
+import java.util.List;
+import java.util.Set;
+
+import static graphql.validation.ValidationErrorType.DuplicateArgumentNames;
+
+@Internal
+public class DeferDirectiveLabel extends AbstractRule {
+    private Set<String> labels = new LinkedHashSet<>();
+    public DeferDirectiveLabel(ValidationContext validationContext, ValidationErrorCollector validationErrorCollector) {
+        super(validationContext, validationErrorCollector);
+    }
+
+    @Override
+    public void checkDirective(Directive directive, List<Node> ancestors) {
+        if (!directive.getName().equals("defer") || directive.getArguments().size() == 0) {
```

Comment:
the string "defer" is repeat many times - can we use a constant from say `graphql.Directives#DEFER`